Form/ChoiceList/ModelChoiceList.php

```diff
@@ -20,7 +20,18 @@
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
 use Symfony\Component\PropertyAccess\PropertyPath;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\ModelChoiceList class is deprecated since SF 2.7'.
```

Comment:
> since SF 2.7

The deprecation is wrong. You are deprecated a Sonata code, note a Symfony code.

Please review guidelines about deprecations: https://github.com/sonata-project/SonataAdminBundle/blob/3.x/CONTRIBUTING.md#the-base-branch